roles/test_operator/defaults/main.yml

```diff
@@ -31,6 +31,16 @@ cifmw_test_operator_tempest_container: openstack-tempest
 cifmw_test_operator_tempest_image: "{{ cifmw_test_operator_tempest_registry }}/{{ cifmw_test_operator_tempest_namespace }}/{{ cifmw_test_operator_tempest_container }}"
 cifmw_test_operator_tempest_image_tag: current-podified
 
+# Enabling SRBAC by default, in jobs where this does not make sense should be turned off explicitly
+cifmw_tempest_tempestconf_config_defaults:
```

Comment:
maybe a new variable, set to true by default, which enables these settings? Or too much?

Reply:
It maybe depends on how "difficult" we want to make the process of turning this off to be and how much we want to force this? Setting one variable to `false` is lot easier then turning SRBAC for every service individually.

Reply:
@tosky i was thinking about that but what about a case when someone wants to enable srbac only for a specific service and not all of them?

Reply:
secure rbac will be on by default for all operators and im not sure if we will supprot turning it off.

my understanding was we wanted to only support srbac

you could use a CustomServiceConfig to diable it in nova or placement but we do not intend to test that.

form an upstrema point of view your are not ment to mix and match services where you use the new policy in some and not other so this is an all or nothing choice for your cloud.

we are not plannig to provide an easy switch to diable srbac in the contol plane so if you wanted to do that as a cutomer you would have to overried the config of every service.
its technically possible but its intetionally not trivial.
we really dont want cutomers turning this off.

Reply:
exactly, as srbac is on by default for all operators, this is meant to enable srbac in tempest by default = anyone who consumes this role will have srbac set by default in their tempest.conf

the motivation for this comes from this ticket - https://issues.redhat.com/browse/OSPRH-1515